backend/telemetry_core/src/state/node.rs

```diff
@@ -110,6 +114,14 @@ impl Node {
         &self.best
     }
 
+    pub fn hwbench(&self) -> Option<&NodeHwBench> {
+        self.hwbench.as_ref()
+    }
+
+    pub fn replace_hwbench(&mut self, hwbench: NodeHwBench) -> Option<NodeHwBench> {
```

Comment:
just for consistency with the other update methods, how about calling this `update_hwbench`?